pkg/apis/serving/v1alpha1/storage_container_types.go

```diff
@@ -34,6 +34,9 @@ type StorageContainerSpec struct {
 	SupportedUriFormats []SupportedUriFormat `json:"supportedUriFormats" validate:"required"`
 	// +kubebuilder:default="initContainer"
 	WorkloadType WorkloadType `json:"workloadType,omitempty"`
+
+	// +kubebuilder:default=false
+	SupportsMultiModelDownload *bool `json:"supportsMultiModelDownload,omitempty"`
```

Comment:
Is there a need to use a flag to enable the feature?
I mean, if you set multiple models, it means that you want to use the feature.